scipy/optimize/setup.py

```diff
@@ -12,7 +12,8 @@ def configuration(parent_package='', top_path=None):
                                     blas_ilp64_pre_build_hook, combine_dict,
                                     uses_blas64, get_f2py_int64_options)
     from scipy._build_utils.compiler_helper import (
-        set_cxx_flags_hook, set_cxx_flags_clib_hook, set_c_flags_hook)
+        set_cxx_flags_clib_hook, set_c_flags_hook)
+    from distutils.sysconfig import get_python_inc
```

Comment:
I think we  should avoid imports from distutils for new code due to the depracation. 

A non exhaustive internet search did not show me any good alternative though. 
Maybe @rgommers could give a hint?

Reply:
According to https://docs.python.org/3/distutils/apiref.html there is no direct replacement. It's fine to use `    from distutils.sysconfig import get_python_inc` here - we'll throw this code away well before `distutils` will disappear.